libkbfs/init.go

```diff
@@ -354,11 +354,18 @@ func Init(ctx Context, params InitParams, keybaseServiceCn KeybaseServiceCn, onI
 	go func() {
 		_ = <-interruptChan
 
+		close(done)
+
 		if onInterruptFn != nil {
 			onInterruptFn()
+
+			// Keep listening on the signal channel in case unmount fails the first
+			// time and user continues pressing Ctrl-C.
+			for range interruptChan {
```

Comment:
Why isn't this an infinite loop?  I don't see anything that closes `interruptChan`.  Is it a requirement that `onInterruptFn` kills the whole process?  Could use some more comments here.